src/test/java/teammates/test/cases/logic/CoursesLogicTest.java

```diff
@@ -1,7 +1,6 @@
 package teammates.test.cases.logic;
 
 import static teammates.common.util.Const.EOL;
-
```

Comment:
Hmm as these lines are not related to the current issue, they should not be changed at all. You should actually reinsert the newlines you removed between lines 4 to 10.